src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -61,6 +61,7 @@ private Drive(WPI_TalonSRX leftLead,  WPI_TalonSRX leftFollower1,  WPI_TalonSRX
 		
 		this.rightLead = rightLead;
 		initMotor(this.rightLead);
+
```

Comment:
Try to keep the rightLead stuff with the right lead stuff and delete anything unused.

Reply:
What do you mean by this?

Reply:
the set inverted and put the rightlead init and theSetSensorPhase together